test/e2e/common/setup-kubestellar.sh

```diff
@@ -66,6 +65,22 @@ make install-local-chart KUBE_CONTEXT=kind-kubeflex
 cd -
 echo "wds1 created."
 
+:
+: -------------------------------------------------------------------------
+: Run OCM transport controller executable
+:
+cd "${SRC_DIR}/../../.." ## go up to KubeStellar directory
+KUBESTELLAR_DIR="$(pwd)"
+cd ocm-transport-plugin-0.1.0-rc1
+echo "replace github.com/kubestellar/kubestellar => ${KUBESTELLAR_DIR}/" >> go.mod
+make build
```

Comment:
If you want to use exec for now, let's not build them. You should have it already in the release tar

Reply:
that's not possible.
an executable is pointing to a specific version of KubeStellar.
assuming current PR changes transport behavior we want to test *the NEW* behavior. 
for this we must get the source of ocm plugin, make sure it points to *the NEW* code that is currently being tested and build the executable/image.

a pre-build executable or image will not test the new changes of KubeStellar transport.

Reply:
That's the  cycle dependency   . We need to resolve it :(

Reply:
That's a risky requirements. So you are basically saying that all transport plugins will need to be re-built?  I believe this should happen only when we change interfaces and the interfaces should be defined in a different repo.

Reply:
If we are going to use the staggered release answer to the dependency cycle then that says what to do here. The instructions and tests in release X of ks/ks use a release of ks/OTP that existed before X was made. Yes, that means that we can not advance both repos simultaneously with each depending on a change in the other. That would be synchronized releases.

Reply:
Perhaps the problem here is that our discussion of how to handle the dependency cycle has not said enough about testing. If we do staggered releases, how does a contributor to ks/OTP test their work? How does a contributor to ks/ks test their work? If the E2E tests are only in ks/ks then there is no way for a contributor to ks/OTP to do E2E tests of their work. We could move the E2E tests to ks/OTP, but that would just move the problem --- giving a contributor to ks/ks no way to do E2E tests of _their_ work. One, painful, answer would be to have E2E tests in both repos.

Reply:
```
If we are going to use the staggered release answer to the dependency cycle then that says what to do here. The instructions and tests in release X of ks/ks use a release of ks/OTP that existed before X was made. Yes, that means that we can not advance both repos simultaneously with each depending on a change in the other. That would be synchronized releases.
```

right. for testing release v0.20.0 of ks/ks we used otp v0.1.0-rc1.
now these testing were over and release v0.20.0 is done.

then we created ks/otp v0.1.0-rc2, this is also done.

now we need to update back to main to use ks/otp 0.1.0-rc2 so that in the next time we release ks/ks we will have the most up to date ks/otp used in the e2e tests.

Reply:
I think we still need work on #1786